muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/transcriptofrecords/TranscriptofRecordsJsPlugin_fi.properties

```diff
@@ -41,7 +41,10 @@ plugin.records.summary.notifications.title = Ilmoitukset
 plugin.records.yo.title = Ylioppilaskirjoituksesi
 plugin.records.yo.abiStatus.title = ABI-status
 plugin.records.yo.abiStatus.content.finished = ABI-statuksestasi ei puutu kursseja. Voit nyt ilmoittautua YO-kokeeseen.
-plugin.records.yo.button.signUp = Ilmoittaudu Yo-kokeeseen
+plugin.records.yo.button.signUp.active = Ilmoittaudu YO-kokeeseen
+plugin.records.yo.button.signUp.active.title = Ilmoittautuminen p��ttyy {0}
+plugin.records.yo.button.signUp.inactive = Ilmoittautuminen alkaa {0}
+plugin.records.yo.button.signUp.inactive.title = P�iv�m��r� t�n��n: {0}
```

Comment:
Lines 46 and 47 should switch so they match the EN locale files.

Reply:
rather  EN locale order is wrong

Reply:
Was